src/vars/foreman.yml

```diff
@@ -0,0 +1,3 @@
+---
+foreman_admin_username: admin
+foreman_admin_password: "{{ lookup('ansible.builtin.password', '/tmp/passwordfile', chars=['ascii_letters']) }}"
```

Comment:
I think we should make this the default in the role and then force users to of this to supply `changeme` to avoid accidents where an installation ends up with the simple well known password.